### PR TITLE
dotCMS/core#24418 fix styles changes and reorder of item appearance

### LINKED ISSUE #24161: UI - Favorite Page portlet: Add empty state design on contentlets table
### Parent Issue
#22343 

image

design mock - WIP

### LINKED ISSUE #24418: UI - Favorite Page portlet: styles changes and reorder of item appearance
### Parent Issue

#24326 

- Add To Bundle command needs to match same order on Content Search portlet
- Listing Page - Language dropdown should have a min-width of 175px so it does not jump in most cases.
- Listing Page - Search box should be ~ 250px (bigger than the language drop down by default).
- Listing Page - Clicking anywhere on the favorites accordion bar should open the favorites accordion.
- Listing Page - If there are no favorites, the favorites accordion should default to closed.
- Adding a Favorite - the url should be editable (maybe we update the thumbnail when changed?)
- The empty favorites listing is not centered:
image

new design:
image

### PR DESCRIPTION
**Pull request by alfredo-dotcms**
### Proposed Changes
- Add To Bundle command needs to match same order on Content Search portlet
- Listing Page - Language dropdown should have a min-width of 175px so it does not jump in most cases.
- Listing Page - Search box should be ~ 250px (bigger than the language drop down by default).
- Adding a Favorite - the url should be editable (maybe we update the thumbnail when changed?)
- The empty favorites listing is not centered:

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
image

### CODE REVIEW

File: core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-favorite-page/dot-favorite-page.component.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ export class DotFavoritePageComponent implements OnInit, OnDestroy {
                         inode: [formStateData?.inode],
                         thumbnail: [formStateData?.thumbnail, Validators.required],
                         title: [formStateData?.title, Validators.required],
-                        url: [{ value: formStateData?.url, disabled: true }, Validators.required],
+                        url: [formStateData?.url, Validators.required],
                         order: [formStateData?.order, Validators.required],
                         permissions: [formStateData?.permissions]
                     });
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-favorite-page/dot-favorite-page.component.ts:77]: Why the `this.isFormValid$` is inside this subscribe? Looks like it doesn't need anything from this subscribe.

File: core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.html [hunk 2]
```diff
@@ -52,14 +54,12 @@
     </ng-template>
 
     <ng-template #emptyState>
-        <dot-pages-card-empty *ngFor="let item of [1, 2, 3, 4, 5]"></dot-pages-card-empty>
-
         <div class="dot-pages-empty__container">
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.html:57]: Are you deleting the files for the `dot-pags-card-empty` component?

File: core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.scss [hunk 3]
```diff
@@ -95,6 +91,11 @@
     font-size: large;
 }
 
+.dot-pages-empty__content {
+    color: $gray;
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.scss:95]: Merge from maste and check the new vars

**Review by fmontes — CHANGES_REQUESTED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-favorite-page/dot-favorite-page.component.ts (hunk 1 above)
Comment (hmoreras) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-favorite-page/dot-favorite-page.component.ts:77]: or use take 1.

Comment (alfredo-dotcms) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-favorite-page/dot-favorite-page.component.ts:77]: it depends on the variable `this.form` which is defined only after the subscribe is done

File: core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.html (hunk 2 above)
Comment (alfredo-dotcms) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.html:57]: done!

File: core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.scss (hunk 3 above)
Comment (alfredo-dotcms) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.scss:95]: done!

**Review by hmoreras — APPROVED** [review]

**Review by fmontes — APPROVED** [review]
Reassign a Observable inside a subscribe is a red flag, approving this but required refactor.

**Pull request merged**